src/components/Navbar/Navbar.styled.ts

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+
+import { themes } from "../../themes";
+
+export const NavbarContainer = styled.nav`
+	width: 100%;
+	background-color: transparent;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	padding: 1em 2em;
+	position: fixed;
+	top: 0;
+	z-index: 1000;
```

Comment:
z-indexy tez mozemy utrzymywać w theme, tak aby łatwiej było je debugować. 
-1 = netagive
0 = neutral
1 = positive
2, 5, 10, 20, 50, 100, 9999 - to np moga byc notyfikacje, modale, dropdowny, headery itd:)